transformers/tests/utils.py

```diff
@@ -6,18 +6,23 @@
 from transformers.file_utils import _tf_available, _torch_available
```

Comment:
cc @aaugustin

Reply:
@julien-c My first instinct would to add the tokenizer test dependencies in the current test runs in Circle CI rather than setting up new test runs and adding the ability to skip these tests. Perhaps I'm missing a good reason why you did it that way?

I see that you recovered https://github.com/aaugustin/transformers/commit/17582458a30d0791ad4f2e18d7eb020f1dbc117f and made good use of it 👍

Reply:
Added a new build target because we expect to bundle other CJK languages in there in the future, and they come with a bunch of dependencies (system-wide, not just pip wide – cf the Python 2 target here) that are a PITA to have to install before running common tests.

Reply:
Added a new build target because we expect to bundle other CJK languages in there in the future, and they come with a bunch of dependencies (system-wide, not just pip wide – cf the Python 2 target here) that are a PITA to have to install before running common tests.